nx.json

```diff
@@ -215,6 +212,14 @@
         "targetName": "pw-e2e",
         "ciTargetName": "e2e-ci"
       }
+    },
+    {
+      "plugin": "@nx/webpack/plugin",
+      "include": ["graph/**/*"],
```

Comment:
I think we should everything. There are no other `webpack.config.ts/js` right?

Reply:
Nope